extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -1437,18 +1436,31 @@ export const BlockaidWarningModal = ({
   const isAsset = !!address;
 
   const WarningInfoBlock = ({ hasArrow = false }: { hasArrow?: boolean }) => (
-    <Alert
-      placement="inline"
-      variant={isWarning ? "warning" : "error"}
-      title={header}
+    <div
+      className={`ScamAssetWarning__box ${
+        isWarning ? "ScamAssetWarning__box--isWarning" : ""
+      }`}
+      data-testid="BlockaidWarningModal__alert"
     >
-      <BlockaidByLine
-        hasArrow={hasArrow}
-        handleClick={() => setIsModalActive(true)}
-        requestId={requestId}
-        address={address}
-      />
-    </Alert>
+      <div className="ScamAssetWarning__box__content">
```

Comment:
Since this is a temporary fix (we should be able to use the <Alert> component in the future), I'm re-using the styles we use on the other Blockaid warnings